src/store/actions/performNextAction/index.ts

```diff
@@ -44,6 +44,11 @@ export const performNextAction = async (
     updates = { error: e.toString() };
   }
   if (updates) {
+    if (!updates.error) {
+      // if no error accured overwrite previous errors in history
+      updates.error = null;
```

Comment:
type of error is `error?: string;`
Which I believe means `null` is not valid. Has to be undefined

Reply:
Maybe you can move this to the mutation instead.

Reply:
https://github.com/liquality/chainify/blob/dev/packages/types/lib/index.ts#L19 this can be used i believe